features/dynamic-bundle-discounts.md

```diff
@@ -15,6 +16,20 @@ This documentation explains the process of setting up **Nosto - Dynamic Bundles*
 
 ### Nosto Dynamic Bundle template
 
+Nosto bundle templates needs the following mandatory variables (name intact) to be configured for proper functioning:
+1. discount type - Defines the type of discount offered (percent/dollar)
```

Comment:
`percent/amount`  might be a better terminology

Reply:
`dollar` is used by shopify. `amount` not supported